spoolman/database/database.py

```diff
@@ -183,6 +186,15 @@ async def _backup_task() -> Optional[Path]:
     return __db.backup_and_rotate(env.get_backups_dir(), num_backups=5)
 
 
+async def _metrics() -> None:
+    """Create some useful prometheus metrics"""
+    logger.info("Start metrics collection")
+    async with get_session() as session:
```

Comment:
Change this to `async for session in get_db_session():` and you don't need to create your own get_session function.